src/libprojectM/Audio/WaveformAligner.hpp

```diff
@@ -0,0 +1,55 @@
+/**
+ * @file WaveformAligner.hpp
+ * @brief Mip-based waveform alignment algorithm
```

Comment:
doc is duplicated

Reply:
Yeah... Will just keep the class doc, and only add a "brief" description for the file.